src/helpers.js

```diff
@@ -105,6 +112,22 @@ function populateHash(source, target = {}) {
   return target
 }
 
+/**
+ * Merge `source` into `target` by merging nested dictionaries.
+ */
+function transferValues(source, target) {
+  for (let key in source) {
+    const sourceValue = source[key]
+    const targetValue = target[key]
+    if (typeof sourceValue === 'object' && typeof targetValue === 'object' && !Array.isArray(sourceValue)) {
```

Comment:
Can you please break this on multiple line to avoid line above 80 char?

```js
if (
  typeof sourceValue === 'object' && 
  typeof targetValue === 'object' && 
  !Array.isArray(sourceValue)
) {
}
```